exercise_10/transformCollectionToString.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Write JS function "transformCollectionToString" to transform collection into string
+    - if array has elements, then return array as string, joined by '|' (pipe character)
+    - otherwise, return an empty string
+ * input: array (collection) 
+ * output: string
+ */
+/**
+ * Transforms a collection into a string.
+ * @param {string[]} collection - The collection to be transformed.
+ * @returns {string} - The transformed collection.
+ */
+/**
+ * CASE #1
+ * input: ['apple', 'grapes', 'strawberries']
+ * output: "apple | grapes | strawberries"
+ */
+/**
+ * CASE #2
+ * input: []
+ * output: ""
+ */
+import checkCollectionHasElements from "../exercise_06/checkCollectionHasElements.js";
+
+const transformCollectionToString = function (collection) {
+  if (checkCollectionHasElements(collection)) {
+    return collection.join(" | ");
+  }
+  return "";
+};
+export default transformCollectionToString;
+
+/**
+ * 
+const transformCollectionToString = function (collection) {
```

Comment:
Nice ternary try.

You can directly apply ternary logic to returned value. Here you have an example

```js
const foo = () => {
  const bar = true
  return bar ? 'OK' : 'KO'
}

const output = foo()
console.log(output) // OK
```

FIX: remove commented code